lib/scarlet/bot.rb

```diff
@@ -10,6 +10,20 @@
 # Our main module, namespacing all of our classes. It is used as a singleton,
 # offering a limited few of methods to start or stop Scarlet.
 class Scarlet
+  module Signals
```

Comment:
I don't think we need an extra class for this, just call `Process.kill name, Process.pid` directly.